client/src/pages/admin/MergeLocations.js

```diff
@@ -0,0 +1,516 @@
+              setFieldValue("lng", location.lng)
+            })}
+          />
+          {map}
+          <div className={"merge-loc-fset" + (reversed ? " reversed" : "")}>
+            <ApprovalSteps
+              type={PLANNING_APPROVAL}
+              steps={location.planningApprovalSteps}
+              fieldSetAction={getActionButton(() =>
+                setFieldValue(
+                  "planningApprovalSteps",
+                  location.planningApprovalSteps
+                )
+              )}
+            />
+          </div>
+
+          <div className={"merge-loc-fset" + (reversed ? " reversed" : "")}>
+            <ApprovalSteps
+              type={PUBLICATION_APPROVAL}
+              steps={location.approvalSteps}
+              fieldSetAction={getActionButton(() =>
+                setFieldValue("approvalSteps", location.approvalSteps)
+              )}
+            />
+          </div>
+        </>
+      )}
+    </div>
+  )
+
+  function getActionButton(onClickAction) {
+    return (
+      <small>
+        <Button
+          icon={reversed ? "double-chevron-left" : ""}
+          rightIcon={reversed ? "" : "double-chevron-right"}
+          intent="primary"
+          text="Use value"
+          onClick={onClickAction}
+        />
+      </small>
+    )
+  }
+}
+
+LocationColumn.propTypes = {
+  location: PropTypes.instanceOf(Location),
+  setLocation: PropTypes.func.isRequired,
+  setFieldValue: PropTypes.func.isRequired,
+  reversed: PropTypes.bool,
+  style: PropTypes.object
+}
+
+LocationColumn.defaultProps = {
+  reversed: false
+}
+
+const LocationField = ({ label, value, align, reversed, action }) => {
+  return (
+    <div
+      style={{
+        borderBottom: "1px solid #CCCCCC",
+        display: "flex",
+        flexDirection: reversed ? "row-reverse" : "row",
+        justifyContent: "space-between",
+        alignItems: "center",
+        padding: "8px 0"
+      }}
+    >
+      <div style={{ flex: "1 1 auto" }}>
+        <div
+          style={{
+            fontWeight: "bold",
+            textAlign: align || "center",
+            textDecoration: "underline"
+          }}
+        >
+          {label}
+        </div>
+        <div style={{ textAlign: align || "center" }}>{value}</div>
+      </div>
+      {action}
+    </div>
+  )
+}
+
+LocationField.propTypes = {
+  label: PropTypes.string.isRequired,
+  value: PropTypes.node,
+  align: PropTypes.string,
+  reversed: PropTypes.bool,
+  action: PropTypes.node
+}
+
+const MergeLocations = ({ pageDispatchers }) => {
+  const history = useHistory()
+  const [saveError, setSaveError] = useState(null)
+  const [mergedLocation, setMergedLocation] = useState(new Location())
+  const [location01, setLocation01] = useState()
+  const [location02, setLocation02] = useState()
+
+  const setLocationFn01 = value => {
+    if (location02?.uuid && value?.uuid && location02.uuid === value.uuid) {
+      toast.error("Please select a different location!")
```

Comment:
a toast is a visual notification that is temporary and goes away, typically used to convey an event or change in state. In this case I believe you want to show a state that should remain visible

Reply:
You are absolutely right about a toast being temporary and conveying an event. This is such a case that we just want to notify the user for selecting another location. I think there is no need for this message to remain visible.